apps/lib/blockly/en_us.js

```diff
@@ -40,8 +40,7 @@ Blockly.Msg.CONTROLS_FOREACH_INPUT_INLIST_TAIL = "";
 Blockly.Msg.CONTROLS_FOREACH_INPUT_ITEM = "for each item";
 Blockly.Msg.CONTROLS_FOREACH_TOOLTIP = "For each item in a list, set the variable '%1' to the item, and then do some statements.";
 Blockly.Msg.CONTROLS_FOR_HELPURL = "https://code.google.com/p/blockly/wiki/Loops#count_with";
-Blockly.Msg.CONTROLS_FOR_INPUT_FROM_TO_BY = "from %1 to %2 count by %3";
-Blockly.Msg.CONTROLS_FOR_INPUT_WITH = "for";
+Blockly.Msg.CONTROLS_FOR_INPUT_FROM_TO_BY = "for %1 from %2 to %3 count by %4";
```

Comment:
we should build this as a new string, so we don't carry over the old translations that don't include "for"